src/views/EventViews001.js

```diff
@@ -1,278 +0,0 @@
-/**
```

Comment:
yes, it can. I implemented all endpoints from here. I dropped _keepAlive() functionality, since as I googled, it's unnecessary for server sent events.( i can add it in the future if there would be related issues)